client/human/HumanClientApp.cpp

```diff
@@ -108,6 +108,8 @@ namespace {
         db.Add("UI.keypress-repeat-interval",   UserStringNop("OPTIONS_DB_KEYPRESS_REPEAT_INTERVAL"),   20,     RangedValidator<int>(0, 1000));
         db.Add("UI.mouse-click-repeat-delay",   UserStringNop("OPTIONS_DB_MOUSE_REPEAT_DELAY"),         360,    RangedValidator<int>(0, 1000));
         db.Add("UI.mouse-click-repeat-interval",UserStringNop("OPTIONS_DB_MOUSE_REPEAT_INTERVAL"),      15,     RangedValidator<int>(0, 1000));
+        db.Add("UI.display-timestamp",          UserStringNop("OPTIONS_DB_DISPLAY_TIMESTAMP"),          true,   Validator<bool>());
+        db.Add("UI.display-timestamp-format",   UserStringNop("OPTIONS_DB_DISPLAY_TIMESTAMP_FORMAT"),   std::string("[%d %b %H:%M:%S] "), Validator<std::string>());
```

Comment:
why is this an option? I don't expect players will ever be able to edit it... would probably make more sense in the stringtable.

Reply:
Removed in base PR.